repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/filters/apivalidator/jsonschemacheck/validator.cfg.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<validators xmlns='http://openrepose.org/repose/validator/v1.0'>
+    <validator
+            role="group1"
+            default="false"
+            check-well-formed="true"
+            check-grammars="true"
+            check-elements="true"
+            xsd-engine="SaxonEE"
+            xpath-version="2"
+            check-plain-params="true"
+            enable-pre-process-extension="false"
+            join-xpath-checks="true"
+            check-headers="true">
```

Comment:
The `check-elements`, `xsd-engine`, `xpath-version`, `enable-pre-process-extension`, `join-xpath-checks`, and `check-headers` can all take the defaults for this test unless you planned on adding more.